lib/controllers/autorizacion_controller.js

```diff
@@ -123,3 +123,25 @@ export const deleteById = async (req, res) => {
 
   res.sendStatus(204);
 };
+
+export const registrarIngreso = async (req, res) => {
+  const { id } = req.params;
+
+  const autorizacion = await Autorizacion.findByPk(id, {
+    include: [
+      { model: Usuario, attributes: ['id', 'nombre', 'apellido', 'dni'] },
+    ],
+  });
+
+  if (!autorizacion) {
+    return res
+      .status(404)
+      .send({ error: `No existe una autorización con el id ${id}` });
+  }
+
+  const data = await autorizacion.update({
+    ...autorizacion,
+    fechaHoraIngreso: new Date(),
+  });
```

Comment:
El `update` está preparado para solamente actualizar los atributos que recibe por parámetro, no es necesario pasarle al objeto completo:

```suggestion
  const data = await autorizacion.update({
    fechaHoraIngreso: new Date(),
  });
```